docs/server.md

```diff
@@ -55,6 +55,41 @@ If you are integrating your tool with bloop and want to install and start the se
   <dd><p>Use the server jar or script in the given path</p></dd>
 </dl>
 
+## Global settings for the server
```

Comment:
I wonder if we should change all references to `.jvmopts` in favor of this approach... The build server references mentions `.jvmopts` often.